reana_workflow_controller/k8s.py

```diff
@@ -145,6 +148,43 @@ def add_reana_shared_storage(self):
         self.kubernetes_objects["deployment"].spec.template.spec.volumes = \
             [volume]
 
+    def _build_cvmfs_volume_mount(self, cvmfs_repos):
+        """Build the Volume and VolumeMount necessary to enable CVMFS.
+
+        :param cvmfs_mounts: List of CVMFS repos to make available. They
+            should be part of ``reana_commons.config.CVMFS_REPOSITORIES``.
+        """
+        cvmfs_map = {}
+        volumes = []
+        volume_mounts = []
+        for repo in cvmfs_repos:
+            if repo in CVMFS_REPOSITORIES:
+                cvmfs_map[CVMFS_REPOSITORIES[repo]] = repo
+
+        for repo_name, path in cvmfs_map.items():
+            volume = get_k8s_cvmfs_volume(repo_name)
+            volumes.append(volume)
+            volume_mounts.append(
+                {'name': volume['name'],
+                 'mountPath': '/cvmfs/{}'.format(path),
+                 'readOnly': volume['readOnly']}
+            )
+
+        return volumes, volume_mounts
```

Comment:
This is the piece of code that should be moved to R-Commons, and [reused in RJC](https://github.com/reanahub/reana-job-controller/blob/fb82d77063a3051153f7868c4a2de5a6680893a4/reana_job_controller/kubernetes_job_manager.py#L148-L165).